docs/current/quickstart/snippets/caching/index.mjs

```diff
@@ -0,0 +1,46 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // highlight-start
+  // create a cache volume
+  const nodeCache = client.cacheVolume("node")
+
+  // use a node:16 container
+  // mount the source code directory on the host
+  // at /src in the container
+  // mount the cache and use it to cache dependencies
+  const source = await client.container()
+    .from("node:16")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] } ))
```

Comment:
maybe refactor the `clien.host` call to a different variable so code is more readable?